.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 160,
```

Comment:
Why such a high print width? Any more than 100 can lead to too much code printed on one line in some cases.

Reply:
I concur. This is a mistake that a lot of people do trying to use prettier for the first time. The line width is not a maximum before you warn like for eslint but how most lines are going to look like. I highly recommend 80 to 100 (maximum).

Reply:
100 is plenty to tab based styling. 👍

Reply:
160 was chosen to avoid too many changes in the code base due to prettier adoption. This may be decreased later.